src/components/TheBreadcrumb.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+    <section class="breadcrumb">
+        <slot/>
+    </section>
```

Comment:
it will have something more in the future? I guess it may have at least back button here 😉

Reply:
Yes! I will work on it later :)